app/models/claim_checking_tasks.rb

```diff
@@ -29,6 +29,7 @@ def applicable_task_names
         task_names.delete("employment_contract")
         task_names.delete("employment_start")
         task_names.delete("subject")
+        task_names.delete("teaching_hours")
```

Comment:
not related to your change but think this class could do with some changes as a whole. be great if we could delegate these to some other classes and switchover from a blacklisting to a whitelist approach, i think would make this so much more readable

Reply:
Yeah I agree, I think these should be moved into the policy as they're policy specific, and would match how we handle admin claim rejection reasons